README.adoc

```diff
@@ -92,24 +108,42 @@ then, run the Maven `liberty:start-server` goal to start them in two Open Libert
 mvn liberty:start-server
 ```
 
+// static guide instructions:
+ifndef::cloud-hosted[]
 Make sure that your Zipkin server is running and point your browser to the {inv-url}/localhost[{inv-url}/localhost^] URL. 
-When you visit this endpoint, you make two GET HTTP requests, one to the `system` service and one to the `inventory`
+When you visit this endpoint, you make two HTTP GET requests, one to the `system` service and one to the `inventory`
 service. Both of these requests are configured to be traced, so a new trace will be recorded in Zipkin.
-Visit the {zipkin-url}[{zipkin-url}^] URL or another location where you configured Zipkin to run and sort the traces
-by newest first. Verify that this new trace contains three spans with the following names:
+Visit the {zipkin-url}[{zipkin-url}^] URL or another location where you configured Zipkin to run. 
+Run an empty query and sort the traces by latest start time first. 
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+Make sure that your Zipkin server is running, and run the following curl command:
+```
+curl http://localhost:9081/inventory/systems/localhost
+```
+{: codeblock}
 
-- `get:io.openliberty.guides.inventory.inventoryresource.getpropertiesforhost`
-- `get:io.openliberty.guides.system.systemresource.getproperties`
-- `add() span`
+When you make this curl request, you make two HTTP GET requests, one to the **system** service and 
+one to the **inventory** service. 
+Both of these requests are configured to be traced, so a new trace will be recorded in Zipkin.
```

Comment:
Both of these requests are configured to be traced, so a new trace will be recorded in Zipkin.
--->
Because tracing is configured for both these requests, a new trace is recorded in Zipkin